common-npm-packages/az-blobstorage-provider/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "azp-tasks-az-blobstorage-provider",
-    "version": "3.232.2",
+    "version": "3.232.3",
```

Comment:
Should not the version be 3.235.0?

Reply:
It's not the task, so I'm not sure it makes sense to keep the sprint number as version for common packages.
I'll have a look how we take versioning for other packages and return back

Reply:
ok, seems others follow the same rule as for tasks. I won't break this consistency in this case. Bumped to 3.235.0